.gitignore

```diff
@@ -7,6 +7,7 @@ __pycache__/
 *.so
 
 # Distribution / packaging
```

Comment:
Remove all dependencies from .gitignore, and leave only specific for your project. Don't use random .gitignore from the internet

Reply:
It it gitignore which I need to choose from GitHub :)